src/main/resources/db/change-log.xml

```diff
@@ -57,10 +57,62 @@
         </createTable>
     </changeSet>
 
-<!--    <changeSet id="insert demo pl" author="Anna Książek">-->
-<!--        <insert tableName="play_list">-->
-<!--            <column name="name">Testowa</column>-->
-<!--        </insert>-->
-<!--    </changeSet>-->
+    <changeSet id="add-orderEntity" author="Ania">
+        <createTable tableName="order_t">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true" />
+            </column>
+            <column name="user_id" type="BIGINT">
+                <constraints nullable="false" unique="false" />
+            </column>
+            <column name="delivery_address_id" type="BIGINT">
+                <constraints nullable="true" unique="false" />
+            </column>
+            <column name="date" type="datetime">
+                <constraints nullable="false" unique="false"/>
+            </column>
+            <column name="status" type="VARCHAR(50)">
+                <constraints nullable="false" unique="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="add-itemEntity" author="Ania">
+        <createTable tableName="item">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true" />
+            </column>
+            <column name="order_id" type="BIGINT">
+                <constraints nullable="false" unique="false" />
+            </column>
+            <column name="product_id" type="BIGINT">
+                <constraints nullable="false" unique="false" />
+            </column>
+            <column name="count" type="INT">
+                <constraints nullable="false" unique="false" />
+            </column>
+            <column name="price" type="DECIMAL">
+                <constraints nullable="false" unique="false" />
+            </column>
+        </createTable>
+
+        <createTable tableName="product">
```

Comment:
to nie zadziała! nie ma nowego changest'a

Reply:
coś pomieszałem, zaraz poprawię

Reply:
Poprawione